README.md

```diff
@@ -76,6 +76,20 @@ Add the new set of test inside `test/tests.json` file like -
 
 ```
 
+#### How to run the tests
+1. Set up the python env locally, see [here](#how-to-setup-aand-build) 
```

Comment:
typo I think. it should be `how-to-setup-and-build` (There are two `a` in the above